reconcile/ldap_users.py

```diff
@@ -42,13 +43,11 @@ def run(dry_run, gitlab_project_id=None, thread_pool_size=10):
                        in user_specs if delete]
 
     if not dry_run:
-        instance = queries.get_gitlab_instance()
-        settings = queries.get_app_interface_settings()
-        gl = GitLabApi(instance, project_id=gitlab_project_id,
-                       settings=settings)
+        mr_cli = mr_client_gateway.init(gitlab_project_id=gitlab_project_id)
```

Comment:
yes. beautiful. if we can find a way to use the external ldap we will even be able to move this integration to run on the cluster as a CronJob if we want.